managed-services/webhosting/reference-content/classic-hosting-migration-information.mdx

```diff
@@ -0,0 +1,172 @@
+---
+meta:
+  title: Web Hosting Classic migration - Technical information
+  description: This page provides technical information for users having migrated their Web Hosting Classic to cPanel.
+content:
+  h1: Web Hosting Classic migration - Technical information
+  paragraph: This page provides technical information for users having migrated their Web Hosting Classic to cPanel.
+tags: webhosting classic migration cpanel
+dates:
+  validation: 2024-01-10
+  posted: 2024-01-10
+categories:
+  - webhosting
+---
+
+You can now migrate your Online.net Web Hosting Classic plan to Scaleway's new Web Hosting platform. This migration brings you a better experience and performance. Benefits of the migration include:
+
+- **Enhanced control with cPanel:** cPanel provides an interface that allows for more granular control over your hosting environment. This ensures a straightforward management experience.
+- **Improved performance:** Migrating to cPanel can result in noticeable performance improvements. Expect more efficient resource utilization and faster loading times for your websites and applications.
+- **Up-to-date system:** cPanel's regular updates contribute to an up-to-date and secure hosting environment. This ensures that your system remains resilient in the face of evolving digital challenges.
+- **A wide range of features:** Explore a range of features offered by cPanel, covering file management, security options, email functionality, and database management. These features collectively form a powerful toolkit to address diverse hosting requirements.
+
+You can initiate the migration of your Web Hosting plan directly from your [Dedibox/Online.net console](https://console.online.net/). For further information, refer to our dedicated documentation [How to migrate an Online.net/Dedibox web hosting plan to Scaleway](/managed-services/webhosting/how-to/migrate-webhosting-plan/).
+
+<Concept opened>
+
+## What will be my new hosting plan?
+
+All migrated hosting will transferred to the **migration plan**.
```

Comment:
This sentence doesn't make sense to me, but I'm not sure what it should be? Is there a link to the details of the SCW plan, like showing exactly the price and featureset? Also, another question I would have is how does the user migrate?